setup.py

```diff
@@ -21,13 +21,22 @@
 pkg_path = os.path.join(here, 'python')
 
 class Distribution(_distribution):
+    global_options = _distribution.global_options
+
+    global_options += [
+        ('enable-boost-cmake', None, 'Enable boost-cmake'),
```

Comment:
If the default adds the option, either the default value should be true or the name should be different

Reply:
default is disabling boost-cmake.
```Boost_NO_BOOST_CMAKE=ON```

I wanted this to be a boolean flag and didn't want to check various 'false' flags

Reply:
Oh I see! Since the default is no